app/src/main/res/values/styles.xml

```diff
@@ -30,6 +40,56 @@
         <item name="fontFamily">@font/bai_jamjuree_regular</item>
         <item name="android:textSize">@dimen/onboarding_text_view_content</item>
         <item name="android:height">@dimen/onboarding_text_view_content_height</item>
-        <item name="android:width">@dimen/onboarding_text_view_content_width</item>
+        <item name="android:width">@dimen/heading2_text_view_content_width</item>
+    </style>
+
+    <style name="AuthentificationHeadingTV" parent="OnboardingTV">
+        <item name="android:textColor">@color/authentication_text_color</item>
+        <item name="android:layout_marginStart">@dimen/horizontal_margin</item>
+        <item name="android:layout_marginTop">@dimen/authentication_heading_top_margin</item>
+    </style>
+
+    <style name="AuthentificationHeading2TV" parent="AuthentificationHeadingTV">
+        <item name="fontFamily">@font/bai_jamjuree_regular</item>
+        <item name="android:textSize">@dimen/heading2_text_size</item>
+        <item name="android:layout_marginTop">@dimen/authentication_heading2_top_margin</item>
+        <item name="android:width">@dimen/heading2_text_view_content_width</item>
+        <item name="android:layout_marginStart">@dimen/horizontal_margin</item>
+        <item name="android:alpha">0.6</item>
+    </style>
+
+    <style name="PrimaryTF" parent="Widget.MaterialComponents.TextInputLayout.FilledBox">
+        <item name="fontFamily">@font/bai_jamjuree_regular</item>
+        <item name="android:textColor">@color/authentication_text_color</item>
+        <item name="android:textSize">@dimen/regular_text_size</item>
+        <item name="android:textColorHint">@color/input_box_hint_color</item>
+        <item name="android:layout_marginStart">@dimen/horizontal_margin</item>
+        <item name="android:layout_marginEnd">@dimen/horizontal_margin</item>
+        <item name="android:hint">This is a hint</item>
+        <item name="hintTextColor">@color/input_box_hint_color</item>
+        <item name="endIconMode">custom</item>
```

Comment:
I wouldn't set this as a general style attribute and would rather have each TIL decide for itself, maybe most of the times the default is good enough